astroplan/plots/time_dependent.py

```diff
@@ -101,6 +102,20 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
         Add alternative y-axis on the right side of the figure with target
         altitude. Default is `False`.
 
+    lo_limit : float
+        Lower limit of y-axis airmass range in the plot. Default is `1.0`.
+
+    lo_region : float
+        If set, defines an interval between `lo_limit` and `lo_region`
```

Comment:
kwarg names be double backticks as sphinx tries to linkify the singe ones and with fail with these.